pkg/appconsts/appconsts.go

```diff
@@ -49,21 +49,25 @@ const (
 	// in a continuation sparse share of a sequence.
 	ContinuationSparseShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes
 
-	// DefaultMaxSquareSize is the maximum number of
-	// rows/columns of the original data shares in square layout.
-	// Corresponds to AVAILABLE_DATA_ORIGINAL_SQUARE_MAX in the spec.
-	// 128*128*256 = 4 Megabytes
-	// TODO(ismail): settle on a proper max square
-	// if the square size is larger than this, the block producer will panic
+	// DefaultMaxSquareSize is the maximum original square width. A square size
+	// larger than this will cause the block producer to panic.
```

Comment:
How does the block producer panic?

Reply:
that comment was from before we used prepare and process proposal, https://github.com/celestiaorg/celestia-core/blob/f4cf3df6e222160a1c4a3c66841dbd0ec17eb408/pkg/consts/consts.go#L33, so we can remove it imo

Reply:
Removed references to panic in https://github.com/celestiaorg/celestia-app/pull/1177/commits/51e826de5358542a797890ad9b26bbfe0e6b7c8f